ports/nrf/boards/feather52840/mpconfigboard.mk

```diff
@@ -1,36 +1,17 @@
 MCU_SERIES = m4
 MCU_VARIANT = nrf52
 MCU_SUB_VARIANT = nrf52840
+SD ?= s140
 SOFTDEV_VERSION ?= 6.0.0
 
-LD_FILE = boards/feather52840/bluefruit_nrf52840_s140_6.0.0.ld
-BOOTLOADER_FILENAME = boards/feather52840/bootloader/feather52840_bootloader_6.0.0_s140_single
+BOOT_SETTING_ADDR = 0xFF000
+BOOT_FILE = boards/$(BOARD)/bootloader/$(SOFTDEV_VERSION)/$(BOARD)_bootloader_$(SOFTDEV_VERSION)_s140
```

Comment:
Shouldn't that s140 at the end be $(SD)?

Reply:
it is currently how bootloader bin is named, 6.0.0 is actually the version for bootloader + sd combo. In case bootloader need some fixes, we will make a new version such as 6.0.0r1 not really a good versioning, but it is easy to know which sd version it contains. I will try to figure out a better naming later.